### ISSUE TITLE
Wrong Labels in Last Graph in "The basics of double/debiased machine learning"

### ISSUE THREAD

**Issue by ShreyDixit** [DoubleML/doubleml-docs#68]
### Description of the issue or change proposal

The labels in the legend of the last graph on the [page](https://docs.doubleml.org/stable/guide/basics.html) are in the wrong order. 

![image](https://user-images.githubusercontent.com/31824249/153384079-a7690ce5-6b3f-4ef3-b2bd-7aaa9d2fc16f.png)

I tried running the code to generate the graph on my system but I failed to reproduce it. The graph that I got after running the code was the right one.
![image](https://user-images.githubusercontent.com/31824249/153384846-cf77ad3d-e967-4e54-adb0-e3fc239e89f5.png)

I even tried to compile the code on my system to generate the HTML files but it still generated the right graph.
Below is my system info that I got after running the following code:
```python
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import doubleml; print("DoubleML", doubleml.__version__)
import sklearn; print("Scikit-Learn", sklearn.__version__)
import seaborn; print("Seaborn", seaborn.__version__)
```

```
Linux-5.11.0-41-generic-x86_64-with-glibc2.10
Python 3.8.8 (default, Apr 13 2021, 19:58:26) 
[GCC 7.3.0]
DoubleML 0.4.1
Scikit-Learn 0.24.1
Seaborn 0.11.1
```

**Comment by MalteKurz** [comment]
So the problem is, that you locally get the correct labels / label order but on the deployed page the labels are wrong?

**Comment by ShreyDixit** [comment]
Yes, precisely.

**Comment by MalteKurz** [comment]
Explicitly setting the labels fixes the issue, see #69.

**Comment by MalteKurz** [comment]
The issue was successfully fixed, see the dev version https://docs.doubleml.org/dev/guide/basics.html. I will trigger a deploy to stable...
